src/mmw/js/src/modeling/views.js

```diff
@@ -266,132 +269,173 @@ var ProjectMenuView = Marionette.ItemView.extend({
+var ScenarioDropDownMenuMoreOptionsView = Marionette.ItemView.extend({
+    model: models.ScenarioModel,
+    template: scenarioMenuItemMoreOptionsTmpl,
+    className: "more-options",
+    ui: {
+        share: '[data-action="share"]',
+        duplicate: '[data-action="duplicate"]',
+    },
+    events: {
+        'click @ui.share': 'showShareModal',
+        'click @ui.duplicate': 'duplicateScenario',
+    },
 
-        if (first) {
-            var aoi_census = first.get('aoi_census');
-            this.collection.createNewScenario(aoi_census);
-        } else {
-            this.collection.createNewScenario();
-        }
+    showShareModal: function() {
+        var share = new modalViews.ShareView({
+                model: new modalModels.ShareModel({
+                    text: 'Scenario',
+                    url: window.location.href,
+                    guest: App.user.get('guest'),
+                    is_private: App.currentProject.get('is_private')
+                }),
+                app: App,
+            });
+
+        share.render();
     },
 
-    onScenarioTabClicked: function(e) {
-        var $el = $(e.currentTarget),
-            cid = $el.data('scenario-cid');
+    duplicateScenario: function() {
+        this.model.collection.duplicateScenario(this.model.cid);
+    },
 
-        this.collection.setActiveScenarioByCid(cid);
+    templateHelpers: function() {
+        return {
+            is_new: this.model.isNew()
+        };
     },
 });
 
-// A scenario tab.
-var ScenarioTabPanelView = Marionette.ItemView.extend({
+// The menu item for a scenario in the scenario drop down menu.
+var ScenarioDropDownMenuItemView = Marionette.LayoutView.extend({
     model: models.ScenarioModel,
     tagName: 'li',
-    template: scenarioTabPanelTmpl,
+    template: scenarioMenuItemTmpl,
+    regions: {
+        'moreOptions': '.more-options-region',
+    },
     attributes: {
         role: 'presentation'
     },
 
     modelEvents: {
-        'change': 'render'
-    },
-
-    templateHelpers: function() {
-        var gis_data = this.model.getGisData().model_input,
-            gwlfe = App.currentProject.get('model_package') === models.GWLFE &&
-                    gis_data !== null &&
-                    gis_data !== '{}' &&
-                    gis_data !== '';
-
-        return {
-            gwlfe: gwlfe,
-            csrftoken: csrf.getToken(),
-            gis_data: gis_data,
-            cid: this.model.cid,
-            editable: isEditable(this.model),
-            is_new: this.model.isNew()
-        };
-    },
-
-    onRender: function() {
-        this.$el.toggleClass('active', this.model.get('active'));
+        'change': 'render',
     },
 
     ui: {
-        share: '[data-action="share"]',
+        selectScenario: '[data-action="select"]',
         destroyConfirm: '[data-action="delete"]',
         rename: '[data-action="rename"]',
-        print: '[data-action="print"]',
```

Comment:
Looks vestigial from https://github.com/WikiWatershed/model-my-watershed/commit/c0d911be34625d7d9529062f7b985991efb0aaf4

There may be a few more references as well.